tests/e2e/integration/withdraw.js

```diff
@@ -47,7 +47,7 @@ describe('Test cases for Withdraw Page', () => {
       .get('[data-cy=review-recipient] > span')
       .should('contain', 'ak_wMHNCzQJ4HUL3TZ1fi6nQsHg6TjmHLs1bPXSp8iQ1VmxGNAZ4')
       .get('[data-cy=review-amount]')
-      .should('contain', '0.200 AE')
+      .contains('0.200 AE')
```

Comment:
Fixup this commit to one that makes it necessary to change this to be able to view the context of the change.

Reply:
I am not sure I understand your request. This change does the exact match of the value without the new line.

Reply:
Doesn't this change become needed because of some other changes that you did in this PR?

Reply:
Yes, [data-cy=review-amount] this element contains the value but introduces a new line.

Reply:
The idea is simple, if I find this commit through blame I need to switch to the previous one to understand why you did it. You can help me a bit by squashing this commit into a previous one, because separatelly it doesn't makes sence. This particular case is not important, but it is important to understand the idea of splitting changes into commits.